index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Mobile Setup</title>
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;800&display=swap" rel="stylesheet">
+</head>
+<body>
+  <header class="container">
+    <section class="nav-bar">   
+      <a class="logo" href="">My Logo</a> 
+      <img class="union" src="/images/Union.png" alt="">   
+   </section> 
+  <main class="content">
+     <h1>I'm Shadrach<br/>
+    Glad to see<br>
+     you!
+     </h1>
+  </main>
+  <article class="second-text">
+     <p>I'm a software developer! I can help you<br>build a product feature or website. Look<br>
+           through some of my work and<br>experience! If you like what you see and<br>have a project 
+            you need coded, don't<br>hesitate to contact me.
+     </p> 
+        <a class="let-connect">LET'S CONNECT </a><br>
+      <sub-article class="icons-social">
+        <a class="icon icon-twitter" href="">
+          <img src="/images/Twitter.png" alt=""></a>
+        <a class="icon icon-linkedin" href=""> 
+          <img src="/images/Linkedin.png" alt=""></a> 
+        <a class="icon icon-medium" href=""> <img src="/images/Vector4.png" alt=""/></a>
+        <a class="icon icon-github" href=""> <img src="/images/Github.png" alt=""/><a>
+        <a class="icon icon-iOS" href=""> <img src="/images/iOS.png" alt=""></a>
+      </sub-article>
```

Comment:
- [ ] Since this is a "List of social media icons" it would be great if instead of using `<sub-article>` you use something like `<ul>` to tell the browser you're going to display a list of elements, in this case, social media links ⭐. Please, change it! 🛠️ 

- [ ] Remember to add your social media links to the `<a>` tags! so people who navigate through your page can click them and connect with you! 😄  

- [ ] Kindly include alternative text `alt` for the icons, so screen readers can identify them and your site becomes more accessible. 🤓